Functions/BIDS/xASL_bids_MergeNifti.m

```diff
@@ -346,14 +347,87 @@
 end
 
 
+%% ===========================================================================================================
+%% ===========================================================================================================
+function pathOut = xASL_bids_MergeNifti_PhilipsMultiTE(NiftiPaths)
```

Comment:
You know for sure that this is specific for multi-TE Philips data, not just in general for Philips data with other parameter vectors?

Reply:
Yes. A very specific Philips thing, deeply connected to the PAR/REC format.

Reply:
Sorry. Yes - specific to multi-TE Philips. The standard Philips merging could be done with the general function.

Though it's probably not necessary to keep multi-TE in the name. I have changed that.

Now I explain that we focus on Philips sequences and multi-TE specifically. Still, this function is supposed to deal with cases that cannot be fixed by general merging. And currently, this is only the multi-TE case. But for future generalizibility, I have removed the multi-TE mention from the function name.

See [here](https://github.com/ExploreASL/ExploreASL/pull/1556/commits/b2b5a5d7c49330706e127bca6b2ac3e490396021)